polaris-core/src/main/java/org/apache/polaris/core/persistence/pagination/DonePageToken.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.core.persistence.pagination;
+
+import java.util.List;
+
+public class DonePageToken extends PageToken {
+
+  public DonePageToken() {}
+
+  @Override
+  public String toString() {
+    return null;
```

Comment:
nit: This is probably a case where we may want to have `toString() == "DONE"`, but `toStringExternal() == null`.

Reply:
Oh, yea... good point. I shied away from `toTokenString` initially because I wanted to avoid accidental use of `toString` and the divergence of the two methods, but I think you're right that in this case we want divergence. Fixed.